recipes/aimodelshare/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "aimodelshare" %}
+{% set version = "0.0.94" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 36b35968e0605fa57610eee39b568a491666ed4d76b5287ea8d276fb83294dd4
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - astunparse ==1.6.3
+    - boto3 ==1.18.2
+    - botocore ==1.21.2
+    - docker-py ==5.0.0
+    - keras2onnx >=1.7.0
+    - onnx >=1.9.0
+    - onnxconverter-common >=1.7.0
+    - onnxmltools >=1.6.1
+    - onnxruntime >=1.7.0
+    - pyjwt ==2.2.0
+    - pympler ==0.9
+    - python >=3.6
+    - scikit-learn ==0.24.2
+    - seaborn >=0.11.2
+    - shortuuid >=1.0.8
+    - skl2onnx >=1.8.0
+    - tensorflow >=2.5.0
+    - tf2onnx
+    - pytorch >=1.8.1
+    - urllib3 ==1.25.11
+    - wget ==1.20.3
+    - xgboost >=0.90
+    - dill
+
+test:
+  imports:
+    - aimodelshare
+    - aimodelshare.custom_approach
+    - aimodelshare.main
+    - aimodelshare.sam
+
+about:
+  home: "https://www.modelshare.org"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: "Deploy locally saved machine learning models to a live rest API and web-dashboard.  Share it with the world via modelshare.org"
+  doc_url: https://github.com/AIModelShare/aimodelshare/blob/master/README.md
+  dev_url: https://github.com/AIModelShare/aimodelshare
+
+extra:
+  recipe-maintainers:
+    - aimodelshare
```

Comment:
We need some confirmation from them that they're willing to be a maintainer.

Reply:
@mikedparrott mike we need your confirmation. thanks!

Reply:
Yes.  Thanks, I confirm that I will be the maintainer of aimodelshare.